util/ranger/ranger_test.go

```diff
@@ -660,6 +660,13 @@ func (s *testRangerSuite) TestIndexRangeForUnsignedInt(c *C) {
 			filterConds: "[]",
 			resultStr:   `[(NULL,1) (2,9223372036854775810) (9223372036854775810,+inf]]`,
 		},
+		{
+			indexPos:    0,
+			exprStr:     `a >= -2`,
```

Comment:
If `a == -2`, we'll get `[eq(test.t.a), 0]`? 
Does it seem to be wrong?

Reply:
no
If a == -2, we'll get [eq(test.t.a), -2]
convertPoint function only affect ranges